modules/core/include/opencv2/core/bindings_utils.hpp

```diff
@@ -144,7 +144,10 @@ AsyncArray testAsyncException()
     return p.getArrayResult();
 }
 
+namespace fs {
+    CV_EXPORTS_AS(getCacheDirectory) cv::String getCacheDirectoryForDownloads();
```

Comment:
why we should rename that?

This would cause confusion between C++ and Python `getCacheDirectory()` functions.